### ISSUE TITLE
Get Mapped Result from Hilbert Curve

### ISSUE THREAD

**Issue by shawnFuu** [galtay/hilbertcurve#46]
Hi, Good work! I am wondering how I can get the mapped result from n-dimensional data point alongside Hilbert Curve. As far as I know, if I use `distance_from_point`, I only get the index of the data instead of mapped value. If you have any advice, that will be very helpful.
Looking forward to your early reply!

**Comment by galtay** [comment]
Can you rephrase the question? I'm not sure exactly what you're asking for. The package converts back and forth between distance along a curve and the discrete coordinates in n-dim space. It sounds like maybe you want to take an arbitrary point in n-dim space and find the closest discrete coordinates? If so, then you can bound each dimension and then do a check on the remaining 2^n discrete points.

**Comment by shawnFuu** [comment]
> Can you rephrase the question? I'm not sure exactly what you're asking for. The package converts back and forth between distance along a curve and the discrete coordinates in n-dim space. It sounds like maybe you want to take an arbitrary point in n-dim space and find the closest discrete coordinates? If so, then you can bound each dimension and then do a check on the remaining 2^n discrete points.

So kind of you to give a reply. I have found that i misunderstood the function. Now I have solved the issue. Thank you so much.
